### DISCUSSION TITLE
Add generic OAuth2/OIDC authentication

### DISCUSSION THREAD

**Discussion by lucasfcnunes** [getredash/redash#6562] [Ideas]
# No generic OAuth2 (only Google) nor OIDC mentions

https://redash.io/help/user-guide/users/authentication-options

# References

https://openid.net/developers/how-connect-works/

**Comment by guidopetri** [comment]
Seems reasonable to me, though likely too complicated for me to digest in a quick skim through that reference link. I think we'd welcome a PR implementing this.

**Comment by palash247** [comment]
Created a WIP PR #6783 with steps to achieve this. Seeking feedback for the same.
